cmd/glasskube/cmd/bootstrap.go

```diff
@@ -31,6 +33,30 @@ var bootstrapCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg, _ := cliutils.RequireConfig(config.Kubeconfig)
 		client := bootstrap.NewBootstrapClient(cfg)
+
+		installedVersion, err := clientutils.GetPackageOperatorVersion(cmd.Context())
+		if err != nil {
+			IsBootstrapped, err := bootstrap.IsBootstrapped(cmd.Context(), cfg)
+			if err != nil && !IsBootstrapped {
+				fmt.Printf("error : %v\n", err)
+				return
+			}
+			return
+		}
+
+		var desiredVersion string
+		if bootstrapCmdOptions.url == "" {
+			desiredVersion = config.Version
+		} else {
+			desiredVersion = bootstrapCmdOptions.url
+		}
+		if !semver.IsUpgradable(installedVersion, desiredVersion) {
+			if !cliutils.YesNoPrompt("Do you want to downgrade the package-operator", false) {
```

Comment:
In this prompt, please also print the information about the version conflict, it should look like this:

"Glasskube is already installed in this cluster in the newer version %v. You are about to install version %v. This could lead to a broken cluster! 
Are you sure that you want to downgrade glasskube in this cluster?"